exec/api/src/main/java/org/jboss/windup/exec/configuration/options/OutputPathOption.java

```diff
@@ -81,15 +86,14 @@ public static ValidationResult validateInputAndOutputPath(Path inputPath, Path o
     public static ValidationResult validateInputsAndOutputPaths(Collection inputPaths, Path outputPath)
     {
 
+        inputPaths.stream().forEach(
+                input -> LOG.log(Level.INFO, "[Jandepora] input.toString())")
+        );
+
         if (inputPaths == null)
         {
             return new ValidationResult(ValidationResult.Level.ERROR, "Input path must be specified.");
         }
```

Comment:
Could you please double-check this block of code? There is a `if NULL` condition AFTER the variable is actually used. If the `inputPaths` is NULL then a NullPointException will appear in line 89 since it is used before it was checked.